components/sysutils/net-snmp/patches/076.21442153.sysconf.patch

```diff
@@ -0,0 +1,23 @@
+Solaris specific patch, not suitable for upstream.
+--- net-snmp-5.8.old/man/snmptrapd.conf.5.def	2018-07-16 07:33:40.000000000 +0000
++++ net-snmp-5.8/man/snmptrapd.conf.5.def	2019-03-08 05:22:34.480553145 +0000
+@@ -298,7 +298,7 @@
+ The default in each case is for the feature to be turned off, so these
+ directives are typically only used to enable the appropriate behaviour.
+ .SH FILES
+-SYSCONFDIR/snmp/snmptrapd.conf
++DATADIR/snmp/snmptrapd.conf
```

Comment:
Is snmptrapd.conf moved out of /etc?

Reply:
No. If used it should be in /etc/net-snmp/snmp
sysconfdir = datadir

CONFIGURE_OPTIONS  += --sysconfdir=$(ETCDIR)/net-snmp
CONFIGURE_OPTIONS  += --with-mibdirs=$(ETCDIR)/net-snmp/snmp/mibs
CONFIGURE_OPTIONS  += --datadir=$(ETCDIR)/net-snmp